### PR TITLE
Release 0.99.0

### PR DESCRIPTION
**Pull request by bwoebi**
0.99 soon will overflow to 1.0 :-)

### CODE REVIEW

**Review by PROFeNoM — APPROVED** [review]
LGTM, with respect to the [Milestone PR List](https://github.com/DataDog/dd-trace-php/milestone/166?closed=1)

File: package.xml [hunk 1]
```diff
@@ -62,77 +62,59 @@
...
-- Mark config as immutable and interned to avoid refcounting race conditions #2516
-
 ## Tracer
-This release includes Windows dlls for the tracer starting with PHP 7.2. #2230
 
-Windows support is in beta. We recommend carefully deploying the tracer to staging environments and testing it there before putting it to production. We are looking towards stabilizing Windows quickly, so any feedback and issue reports are welcome.
+Windows support is officially GA (generally available) and considered stable.
+
+Some performance work has been undertaken to primarily reduce the overhead of telemetry reporting.
+The next release will be version 1.0 and remove deprecated and some unused code and APIs.
```
Comment (PROFeNoM) [inline package.xml:70]: 💭 Should we also emphasize the newly added FrankedPHP Support?

Comment (pierotibou) [reply package.xml:70]: Let's not say the next release but maybe "we will soon release...". We can add thé branch for people that are curious

File: package.xml [hunk 2]
```diff
@@ -62,77 +62,59 @@
...
-- Handle Apache graceful restarts more accurately #2483
-- Mark config as immutable and interned to avoid refcounting race conditions #2516
-
 ## Tracer
-This release includes Windows dlls for the tracer starting with PHP 7.2. #2230
 
-Windows support is in beta. We recommend carefully deploying the tracer to staging environments and testing it there before putting it to production. We are looking towards stabilizing Windows quickly, so any feedback and issue reports are welcome.
+Windows support is officially GA (generally available) and considered stable.
+
+Some performance work has been undertaken to primarily reduce the overhead of telemetry reporting.
```
Comment (pierotibou) [inline package.xml:69]: Can WE share a number with cautious wording?

File: package.xml [hunk 3]
```diff
@@ -62,77 +62,59 @@
...
         <![CDATA[
-### Fixes across all extensions
-- Handle Apache graceful restarts more accurately #2483
-- Mark config as immutable and interned to avoid refcounting race conditions #2516
-
 ## Tracer
-This release includes Windows dlls for the tracer starting with PHP 7.2. #2230
 
-Windows support is in beta. We recommend carefully deploying the tracer to staging environments and testing it there before putting it to production. We are looking towards stabilizing Windows quickly, so any feedback and issue reports are welcome.
+Windows support is officially GA (generally available) and considered stable.
```
Comment (pierotibou) [inline package.xml:67]:
```suggestion
Windows x64 support is officially GA (generally available).
```

File: package.xml (hunk 1 above)
Comment (pierotibou) [reply package.xml:70]: Let's make sure we close the swoole(laravel octane) issue and FRs on Tuesday

File: package.xml (hunk 2 above)
Comment (bwoebi) [reply package.xml:69]: It depends on things, like composer.json size and depends on what your app is actually doing, like if there would be a lot of logs or not if logging would be fully turned on.

For the changes on the ddtrace-1.0.0 branch we probably can have more concrete perf numbers as it's a rather overall startup improvement.

File: package.xml (hunk 1 above)
Comment (pierotibou) [reply package.xml]: As we said we need one month between com and release, that's a possibility.
That's said we could issue a patch release if you want to merge, as indeed that's going to be easier. I'd ask the other teams if they are ok with it. And put Tushar in the loop for visibility

File: package.xml [hunk 4]
```diff
@@ -62,77 +62,59 @@
...
+- enabled allocation profiling for PHP ZTS #2506
+- report PHP build (`nts-ndebug` vs `zts-ndebug`) #2536
 
 ### Internal
-- Update dependencies and pin `ahash` #2479
-- Fix clippy::single-match #2482
-- Update to libdatadog 6 #2513
-- Refactor: use SystemSettings in locals #2487
-- Fix accidentally skipped tests #2517
+- add tests for exception profiling in ZTS builds #2562
```
Comment (morrisonlevi) [inline package.xml:105]: This one was merged after this PR was opened:

```suggestion
- add tests for exception profiling in ZTS builds #2562
- update to libdatadog v7 #2605
```

**Pull request merged**